### PR TITLE
Touch-up some locale logic

### PR DESCRIPTION
**Pull request by jbukuts**
Fixes | Relates to **[issue link]**

Content changes:

- N/A

Code changes:

- Dates are now formatted for the currently selected language
- Page header now reflects this with its own translation
- Added logic for nested routes to display recently updated and total time on site homepage
- Small clean-up of locale logic

Testing:
- To see locale changes simply switch between langs on the homepage or any content page and see the date formatting change
- To see the logic for nested routes look at the watsonx tile on the home page and compare to main branch. It should now have data whereas the main branch will not display anything.

### CODE REVIEW

File: i18n.ts [hunk 1]
```diff
@@ -5,10 +5,33 @@ import * as translations from './translations';
 export interface SupportedLanguage {
   id: string;
   label: string;
+  dateFormat: string;
 }
 
+export type Lang = 'en' | 'es';
+
+export const DEFAULT_LOCALE: Lang = 'es';
```
Comment (ridgill) [inline i18n.ts:13]: Should we make english the default?  That will arguably be the most used language for the labs.

**Comment by ridgill** [comment]
Looks good to me, just the suggestion that we use default lang of 'en'

File: i18n.ts (hunk 1 above)
Comment (jbukuts) [reply i18n.ts]: You are correct. My mistake.

**Review by ridgill — APPROVED** [review]
LGTM, thanks!

**Pull request merged**